packages/psss/src/containers/Tables/CountryReportTable.js

```diff
@@ -117,11 +127,14 @@ export const CountryReportTableComponent = React.memo(
             Edit
           </GreyOutlinedButton>
         </FlexSpaceBetween>
-        {tableStatus === TABLE_STATUSES.EDITABLE && (
+        {tableStatus === TABLE_STATUSES.EDITABLE ? (
           <Alert severity="error" variant="standard">
-            Updating aggregated data will be the source of truth. All individual Sentinel data will
-            be ignored
+            Updating country level data manually: all individual sentinel site data will be ignored
           </Alert>
+        ) : (
+          <GreyAlert severity="info" icon={<InfoIcon fontSize="inherit" />}>
+            Country level data has been manually edited, sentinel data will not be used.
```

Comment:
We'll only want to show this under a specific condition, which I guess we'll have to return as a boolean from the back end: has the country level data actually been manually edited

Reply:
Can you add that in to the fake api?